addons/sale_timesheet/views/hr_timesheet_views.xml

```diff
@@ -56,6 +56,8 @@
         <field name="inherit_id" ref="hr_timesheet.hr_timesheet_line_form"/>
         <field name="arch" type="xml">
             <xpath expr="//field[@name='task_id']" position="after">
+				<field name="user_id" invisible="1" groups="sales_team.group_sale_salesman"/>
+				<field name="employee_id" invisible="1" groups="sales_team.group_sale_salesman"/>
```

Comment:
is it really needed to add them on the view? 🤔

Reply:
Not really, just wasn't sure if removal from view was intentional because I couldn't to pinpoint exactly when the bug was introduced, but it was somewhere between the refactoring of https://github.com/odoo/odoo/pull/117748 and the switch to the milk theme.

Using the same fix used in https://github.com/odoo/odoo/pull/133478 is adequate to solve this issue.